syntax-highlighting-code-block.php

```diff
@@ -238,7 +238,7 @@ function render_block( $attributes, $content ) {
 	$transient_key = 'syntax-highlighting-code-block-' . md5( $attributes['showLines'] . $attributes['language'] . $matches['code'] ) . '-v' . PLUGIN_VERSION;
 	$highlighted   = get_transient( $transient_key );
 
-	if ( $highlighted && isset( $highlighted['code'] ) ) {
+	if ( ! DEVELOPMENT_MODE && $highlighted && isset( $highlighted['code'] ) ) {
```

Comment:
Why the `! DEVELOPMENT_MODE` check?

Reply:
I might have been doing something wrong but I believe I kept getting cached results while I was developing on the plugin. So I figured I'd disable transients in dev mode. Is there a better way of handling this?